community/ndp/v1-docs/src/docs/dev/serialization.asciidoc

```diff
@@ -406,12 +406,30 @@ Relationship (signature=0x52) {    /* signature: 'R' */
 
 [[ndp-value-pathstruct]]
 ==== Path
-A Path consists of a list of alternating nodes and relationships, always starting and ending with a node. The general serialised structure is as follows:
+A Path represents a walk through a graph.
+Its representation consists of a set of distinct nodes included within the path, a similar set of distinct relationships and a sequence describing the shape of the path.
```

Comment:
I'd love to have this expanded in a follow-up commit, since this section in the docs is the only public formal documentation we provide for NDP. As a suggestion, how about we expand a bit to explain how the sequence is interpreted, and then add a path example in the examples section of the NDP docs, and link to that example from here?